TOCropViewController/Views/TOCropView.m

```diff
@@ -253,6 +254,36 @@ - (void)layoutInitialImage
     frame.origin.x = bounds.origin.x + floorf((CGRectGetWidth(bounds) - frame.size.width) * 0.5f);
     frame.origin.y = bounds.origin.y + floorf((CGRectGetHeight(bounds) - frame.size.height) * 0.5f);
     self.cropBoxFrame = frame;
+    
+    if (self.applyInitialRotatedAngle) {
+        // Get croppedFrame before rotate.
+        // After getting croppedFrame before rotate, rotate.
+        CGRect beforeRotatedFrame = self.initialCroppedImageFrame;
+        switch (self.initialRotatedAngle) {
+            case 90:
+            case -270:
+                beforeRotatedFrame.origin.x = self.initialCroppedImageFrame.origin.y;
+                beforeRotatedFrame.origin.y = self.imageSize.height - self.initialCroppedImageFrame.size.width - self.initialCroppedImageFrame.origin.x;
+                beforeRotatedFrame.size = CGSizeMake(self.initialCroppedImageFrame.size.height, self.initialCroppedImageFrame.size.width);
+                break;
+            case 180:
+            case -180:
+                beforeRotatedFrame.origin.x = self.imageSize.width - self.initialCroppedImageFrame.size.width - self.initialCroppedImageFrame.origin.x;
+                beforeRotatedFrame.origin.y = self.imageSize.height - self.initialCroppedImageFrame.size.height - self.initialCroppedImageFrame.origin.y;
+                beforeRotatedFrame.size = self.initialCroppedImageFrame.size;
+                break;
+            case 270:
+            case -90:
+                beforeRotatedFrame.origin.x = self.imageSize.width - self.initialCroppedImageFrame.size.height - self.initialCroppedImageFrame.origin.y;
+                beforeRotatedFrame.origin.y = self.initialCroppedImageFrame.origin.x;
+                beforeRotatedFrame.size = CGSizeMake(self.initialCroppedImageFrame.size.height, self.initialCroppedImageFrame.size.width);
+                break;
+            default:
+                break;
+        }
+        self.initialCroppedImageFrame = beforeRotatedFrame;
+    }
+
```

Comment:
I applied below steps firstly.
1. rotate 2. crop
Because the delegate returns frame after rotate an crop.
But there are some bugs. wrong location.

So I tried changing steps.
1. crop (before rotate frame)
2. rotate

it's working well.